docs/.vitepress/components/SnippetToggler.vue

```diff
@@ -85,66 +76,37 @@ onBeforeMount(() => {
 	height: 40px;
 	display: flex;
 	align-items: center;
-	padding: 0 24px;
+	padding: 24px;
 }
 
-.spacer {
-	flex-grow: 1;
-}
-
-.snippet-toggler-header-label {
-	text-transform: uppercase;
-	font-size: 12px;
-	font-weight: 600;
-}
-
-.snippet-toggler-header-lang-container {
+.buttons {
 	display: flex;
-	align-items: center;
+	gap: 0.5em;
 }
 
-.snippet-toggler-header-lang-container:hover {
-	color: var(--vp-snippet-toggler-lang-hover);
+.button {
+	padding: 0.25em 0.75em;
+	color: var(--vp-c-gray);
 }
 
-.snippet-toggler-header-lang {
-	background-color: transparent;
-	text-align: right;
-	border: 0;
-	border-color: transparent;
-	padding: 0;
-	font-family: inherit;
-	color: inherit;
-	appearance: none;
-	line-height: inherit;
-	position: relative;
-	font-size: 12px;
-	padding-inline-end: 20px;
-}
-.snippet-toggler-header-lang option {
+.button.active {
 	color: var(--vp-c-black);
+	background: var(--vp-c-mute);
+	border-radius: var(--rounded-lg);
 }
 
-.snippet-toggler-header-lang:focus {
-	outline: none;
+html.dark .snippet-toggler .button {
+	color: var(--vp-c-gray-light-2);
 }
 
-.snippet-toggler-header-lang-arrow {
-	position: absolute;
-	right: 24px;
-	fill: currentColor;
-	user-select: none;
-	pointer-events: none;
+html.dark .snippet-toggler .button.active {
+	color: var(--vp-c-gray-light-4);
 }
 
 .snippet-toggler .content-area :deep(.lang) {
 	display: none;
 }
 
-.snippet-toggler.dark .content-area :deep(.vp-code-dark) {
-	display: block;
-}
-
```

Comment:
This is required for the start page:
<img width="667" alt="Screenshot 2023-09-12 at 12 10 08" src="https://github.com/directus/directus/assets/5363448/a536ead7-c3ad-4de4-a708-7515980072d9">

Reply:
GOTCHA. Didn't know what it was doing there.